lib/messagebird/request_validator.rb

```diff
@@ -71,14 +71,14 @@ def decode_signature(signature)
 
     def validate_url(url, url_hash)
       expected_url_hash = Digest::SHA256.hexdigest url
-      unless JWT::SecurityUtils.secure_compare(expected_url_hash, url_hash)
+      unless secure_compare(expected_url_hash, url_hash)
         raise ValidationError, 'invalid jwt: claim url_hash is invalid'
```

Comment:
should this error message refer to an OpenSSL error now rather than jwt?

Reply:
I don't believe so - the error indicates that the supplied *jwt data* is invalid.

In https://github.com/BloomAndWild/messagebird-ruby-rest-api/blob/master/spec/request_validator_spec.rb you can see that the functionality is checking a series of JWTs (https://github.com/BloomAndWild/messagebird-ruby-rest-api/blob/master/spec/data/webhook_signature_test_data.json) against the rules.

So the intention of the error remains consistent

Reply:
Okay, sorry, I completely misunderstood!